src/SignalR/clients/ts/FunctionalTests/yarn.lock

```diff
@@ -1402,12 +1402,13 @@ karma-edge-launcher@^0.4.2:
   dependencies:
     edge-launcher "1.2.2"
 
-karma-firefox-launcher@^1.3.0:
-  version "1.3.0"
-  resolved "https://registry.yarnpkg.com/karma-firefox-launcher/-/karma-firefox-launcher-1.3.0.tgz#ebcbb1d1ddfada6be900eb8fae25bcf2dcdc8171"
-  integrity sha512-Fi7xPhwrRgr+94BnHX0F5dCl1miIW4RHnzjIGxF8GaIEp7rNqX7LSi7ok63VXs3PS/5MQaQMhGxw+bvD+pibBQ==
+karma-firefox-launcher@^2.1.2:
+  version "2.1.2"
+  resolved "https://registry.yarnpkg.com/karma-firefox-launcher/-/karma-firefox-launcher-2.1.2.tgz#9a38cc783c579a50f3ed2a82b7386186385cfc2d"
```

Comment:
Unrelated to this change. @wtgodbe should we fix the "resolved" section in servicing branches? We fixed them up in 8.0 but didn't downlevel.

Reply:
Does that mean adding .yarnrc files/.npmrc files & getting it to resolve our AzDO feed?

Reply:
I think so, and adding the script that makes sure we don't put a different registry in the lock files